tests/python/gaia-ui-tests/gaiatest/apps/notes/app.py

```diff
@@ -0,0 +1,51 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Nit: `__init__.py` is missing in `tests/python/gaia-ui-tests/gaiatest/apps/notes/`. Then, the test will fail with this error:
`ImportError: No module named notes.app`